src/comfyui-refiners/README.md

```diff
@@ -25,7 +25,13 @@ To easily install the nodes, run the following command:
 comfy node registry-install comfyui-refiners
 ```
 
-You may also download the nodes by cliking the "Download Latest" button and unzipping the content of the archive into you custom_nodes directory.
+To manually install the nodes, you may alternatively do the following:
+```bash
+wget -O comfyui-refiners.zip https://storage.googleapis.com/comfy-registry/finegrain/comfyui-refiners/1.0.3/node.tar.gz
+unzip -d custom_nodes/comfyui-refiners comfyui-refiners.zip
+pip install -r custom_nodes/comfyui-refiners/requirements.txt
```

Comment:
Don't want to add a note / warning about using a venv? (and/or other recommendations)